### DISCUSSION TITLE
YARP as Reverse Proxy without other web server

### DISCUSSION THREAD

**Discussion by santoni1981** [dotnet/yarp#2672] [General] [answered]
Hi, I am interested to know if YARP can be used to replace full-featured web servers such as Microsoft IIS, NGINX or Apache HTTPd to expose my application to the Internet.
I am creating a small web application, and I want to expose the services through YARP as API Gateway without using another web server. Can YARP serve this purpose without security risks?
Thanks.

**Comment by MihaZupan** [comment, marked as answer]
Yes

**Reply by santoni1981** [reply]
> Yes

Thanks
